### PR TITLE
Pension savings timeline

### PR DESCRIPTION
**Pull request by mconnell**
Adds `/en/pensions-and-retirement/pension-savings-timeline`

Deployed to the preview server here: https://preview.test.moneyadviceservice.org.uk/en/pensions-and-retirement/pension-savings-timeline

### CODE REVIEW

File: app/views/retirements/pension_savings/_throughout_working_life_section.html.erb [hunk 1]
```diff
@@ -0,0 +1,102 @@
...
+      <div class="linear">
+        <div class="line-marker line-marker--waypoint">
+          <div class="line-marker__small-outline-dot"></div>
+        </div>
+        <div class="speech-bubble speech-bubble--right">
+          <h3>
+          <%= t('pension_savings_timeline.throughout_working_life.getting_guidance_and_advice.title') %>
+          </h3>
+          <p>
+          <%= t('pension_savings_timeline.throughout_working_life.getting_guidance_and_advice.text') %>
```
Comment (tomas-stefano) [inline app/views/retirements/pension_savings/_throughout_working_life_section.html.erb]: Can you indent this last two, please?

File: app/views/retirements/pension_savings/_throughout_working_life_section.html.erb [hunk 2]
```diff
@@ -0,0 +1,102 @@
...
+        </div>
+        <div class="speech-bubble speech-bubble--right">
+          <h3>
+          <%= t('pension_savings_timeline.throughout_working_life.managing_your_debt.title') %>
+          </h3>
+          <p>
+          <%= t('pension_savings_timeline.throughout_working_life.managing_your_debt.text') %>
+          </p>
+          <p>
+          <%= link_to t('pension_savings_timeline.throughout_working_life.managing_your_debt.link'), t('pension_savings_timeline.throughout_working_life.managing_your_debt.url') %>
```
Comment (tomas-stefano) [inline app/views/retirements/pension_savings/_throughout_working_life_section.html.erb]: Can you indent this last three, please?

File: app/views/retirements/pension_savings/_throughout_working_life_section.html.erb [hunk 3]
```diff
@@ -0,0 +1,102 @@
...
+        </div>
+        <div class="speech-bubble speech-bubble--left">
+          <h3>
+          <%= t('pension_savings_timeline.throughout_working_life.saving_more_into_your_pension.title') %>
+          </h3>
+          <p>
+          <%= t('pension_savings_timeline.throughout_working_life.saving_more_into_your_pension.text') %>
+          </p>
+          <p>
+          <%= link_to t('pension_savings_timeline.throughout_working_life.saving_more_into_your_pension.link'), t('pension_savings_timeline.throughout_working_life.saving_more_into_your_pension.url') %>
```
Comment (tomas-stefano) [inline app/views/retirements/pension_savings/_throughout_working_life_section.html.erb]: Can you indent this last three, please?

File: app/views/retirements/pension_savings/_throughout_working_life_section.html.erb [hunk 4]
```diff
@@ -0,0 +1,102 @@
...
+        </div>
+        <div class="speech-bubble speech-bubble--right">
+          <h3>
+          <%= t('pension_savings_timeline.throughout_working_life.monitoring_your_pension_investments.title') %>
+          </h3>
+          <p>
+          <%= t('pension_savings_timeline.throughout_working_life.monitoring_your_pension_investments.text') %>
+          </p>
+          <p>
+          <%= link_to t('pension_savings_timeline.throughout_working_life.monitoring_your_pension_investments.link'), t('pension_savings_timeline.throughout_working_life.monitoring_your_pension_investments.url') %>
```
Comment (tomas-stefano) [inline app/views/retirements/pension_savings/_throughout_working_life_section.html.erb]: Can you indent this last three, please?

**Comment by slaywell** [comment]
A few front end / CSS comments - drop me a line if you want to discuss?

1. The new icons are visibly broken in IE8 (due to background-size). Also it would be good to implement these either using SVG (with PNG spritesheet fallback) as per the rest of the site - alternatively the circles could be done in pure CSS to save requests.
2. Please can you remove the comments throughout SCSS files (is this still WIP?)?
3. Please can you remove the !important declarations, I don't think they are actually needed? Also we may be able to remove the .line-marker__inline-dot class as I don't think it's used.
4. The whitespace between separate declarations is not consistent with our style in a few places, I'll put an inline comment as an example but refer to other .scss files for examples.

File: app/assets/stylesheets/components/page_specific/retirement_concepts/_line_markers.scss [hunk 5]
```diff
@@ -0,0 +1,43 @@
+.line-marker {
+  position: relative;
+  z-index: 2;
+  left: 10px;
+  @include respond-to($mq-m) {
+    left: 50%;
+  }
+
```
Comment (slaywell) [inline app/assets/stylesheets/components/page_specific/retirement_concepts/_line_markers.scss]: Extra whitespace here

**Comment by tomas-stefano** [comment]
:+1:

**Comment by slaywell** [comment]
Think it would be good to remove the image files that are now in the spritesheet?

**Comment by slaywell** [comment]
:+1:

**Comment by munckymagik** [comment]
Hi @slaywell the images have been removed as requested. We hope you don’t mind but we’re assuming that was your only remaining feedback before a thumb, so going to go ahead and merge as we are under time pressure and aware that you're in a meeting. If you have any further feedback we will happily address it in a new PR.

**Pull request merged**